client/components/widget/query/query-result-data-service.js

```diff
@@ -44,6 +44,8 @@ const WorkQueueService = explorer.components.util.WorkQueueService;
 const WidgetConfig = explorer.models.WidgetConfig;
 
 
+const ERR_UNEXPECTED = 'An unexpected error occurred.';
```

Comment:
We can make this more obvious - "An error occurred when fetching data from %s, but the http error offered no details", where %s is the data provider or datasource name. (e.g. BigQuery or CloudSQL now, "my data source #12" in the future".